src/test/java/com/fasterxml/jackson/databind/ser/TestCustomSerializers.java

```diff
@@ -318,4 +370,28 @@ public void testIssue2475() throws Exception {
         assertEquals(a2q("{'id':'ID-2','set':[]}"),
                 writer.writeValueAsString(new Item2475(new HashSet<String>(), "ID-2")));
     }
+
+    @Test
+    public void testIssue4575() throws Exception {
+        com.fasterxml.jackson.databind.Module module = new SimpleModule() {
+            {
+                setSerializerModifier(new BeanSerializerModifier() {
+                    @Override
+                    public JsonSerializer<?> modifySerializer(
+                        SerializationConfig config, BeanDescription beanDesc, JsonSerializer<?> serializer
+                    ) {
+                        return new NullSerializer4575(config.getTypeFactory(), serializer);
+                    }
+                });
+            }
+        };
+
+        ObjectMapper mapper = jsonMapperBuilder()
```

Comment:
Using builder since "ObjectMapper.registerModule()" removed from 3.0, builder-approach easier to merge forward.